lib/usb/libusbohci_xbox/xid_driver.h

```diff
@@ -36,7 +36,24 @@ typedef struct __attribute__((packed))
     uint16_t wAlternateProductIds[4];
 } xid_descriptor;
 
-typedef struct __attribute__((packed)) 
+//XINPUT defines and struct format from
+//https://docs.microsoft.com/en-us/windows/win32/api/xinput/ns-xinput-xinput_gamepad
```

Comment:
These docs are probably intended for Xbox 360 and up (when MS publicly documented XInput and provided Windows drivers).
Original Xbox doesn't even have "LEFT_SHOULDER" and "RIGHT_SHOULDER" - it uses black and white face-buttons.